CRM/Civicase/Form/Report/BaseExtendedReport.php

```diff
@@ -597,6 +608,7 @@ public function buildQuery($applyLimit = TRUE) {
     $this->from();
     $this->where();
     $this->extendedCustomDataFrom();
+    $this->extendedWhere();
```

Comment:
Well I investigated a bit on this but could not find any other possible reason and looks like that this indeed is the reason for the problem as for neu there are many case roles and each case roles results in one increased join which duplicates the all the contact core and custom fields so I think for neu this could be the reason that the overall result of the query gets so big due to the large number of joins